code-challenges.js

```diff
@@ -13,52 +24,87 @@ var num3 = -7
 // Expected output: "-7 is not divisble by three"
 
 
-
+//Testing
+console.log(divByThree(num1))
+console.log(divByThree(num2))
+console.log(divByThree(num3))
 
 
 // --------------------2) Create a function that takes in the randomNouns variable and returns an array with all the words capitalized.
 
 var randomNouns = ["streetlamp", "potato", "teeth", "conclusion", "nephew", "temperature", "database"]
 // Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew", "Temperature", "Database"]
 
+const capIt = (array) => {
+  return array.map(value => value.charAt(0).toUpperCase() + value.slice(1))
+}
 
-
-
+console.log("streetlamp".charAt(0).toUpperCase() + "streetlamp".slice(1))
+console.log("streetlamp"[1])
+console.log(capIt(randomNouns))
 
 // --------------------3) Create a function that takes in the mixedDataArray array and returns an array with ONLY NUMBERS sorted from least to greatest.
 
 var mixedDataArray = [true, 8, "hello", 90, -8, null, 0, 46, 59, 107, "hey!"]
 // Expected output: [-8, 0, 8, 46, 59, 90, 107]
 
+const onlyNum = (array) => {
+  return array.filter(value => typeof value === "number").sort(function(a,b){return a-b})
+}
 
-
-
+console.log(onlyNum(mixedDataArray))
 
 // --------------------4) Create a function that takes in a string and logs the index of the first vowel.
 // Use the test variables provided.
 
 var vowelTester1 = "learn"
 // Expected output: 1
-var vowelTester1 = "throw"
+var vowelTester2 = "throw"
 // Expected output: 3
 
+const vowelTester = (str) => {
+  let vowels = ["a", "e", "i", "o", "u"]
+  str.split("")
```

Comment:
you actually don't even need to split here! strings can be iterated through by loops just like arrays, isn't that cool ^.^